homeassistant/components/mysensors/handler.py

```diff
@@ -1,93 +1,104 @@
 """Handle MySensors messages."""
+from typing import Dict, List
+
+from mysensors import Message
+
 from homeassistant.core import callback
 from homeassistant.helpers.dispatcher import async_dispatcher_send
+from homeassistant.helpers.typing import HomeAssistantType
 from homeassistant.util import decorator
 
-from .const import CHILD_CALLBACK, MYSENSORS_GATEWAY_READY, NODE_CALLBACK
+from .const import (
+    CHILD_CALLBACK,
+    MYSENSORS_GATEWAY_READY,
+    NODE_CALLBACK,
+    DevId,
+    GatewayId,
+)
 from .device import get_mysensors_devices
 from .helpers import discover_mysensors_platform, validate_set_msg
 
 HANDLERS = decorator.Registry()
 
 
 @HANDLERS.register("set")
-async def handle_set(hass, hass_config, msg):
+async def handle_set(hass, gateway_id: GatewayId, msg: Message) -> None:
```

Comment:
Please type the functions completely if we add typing.
```suggestion
async def handle_set(hass: HomeAssistantType, gateway_id: GatewayId, msg: Message) -> None:
```